_hands-on/singularity/singularity-tutorial_running-installed.md

```diff
@@ -7,14 +7,14 @@ title: Tutorial - Running applications installed as containers
 
 💬 In this tutorial we get familiar with the basic usage of software that has been installed as a container.
 
-💭 Some software on CSC supercomputers have been installed as containers. 
-- In many cases we have tried to "hide" the container behind wrapper scripts so that there is no change in usage from the user perspective.
+💭 Some of the software on the CSC supercomputers has been installed as containers. 
+- In many cases we have tried to "hide" the container behind wrapper scripts so that there is no change in the usage from the user perspective.
 
-💭 On some cases, however, this is not practical, and in these cases there can be some differences to the standard usage as described by the software documentation. 
--Typically, this involves prefixing the commands with `singularity_wrapper exec`. 
--This will also typically be the case for software you install yourself as a container.
+💭 Sometimes, however, this is not practical, and in these cases, there can be some differences compared to the standard usage as described in the software documentation. 
+- Typically, this involves prefixing the commands with `apptainer_exec`. 
```

Comment:
Please change "apptainer_exec" to "apptainer_wrapper exec" 

In the comments, I would add that for most container installations at CSC, we use Tykky as container wrapper tool and hence "apptainer_wrapper exec" is not needed to prefix before the actual command.